Makefile

```diff
@@ -128,15 +128,12 @@ endif
 
 XGO := $(strip $(if $(XGOOS),GOOS=$(XGOOS)) $(if $(XGOARCH),GOARCH=$(XGOARCH)) $(if $(XHOST_TRIPLE),CC=$(CC_PATH) CXX=$(CXX_PATH)) $(GO))
 
-.DEFAULT_GOAL := all
-.PHONY: all
-all: build test lint
```

Comment:
I'm not worried, I just think it's not necessary. If you have PHONY on `build` (because there's a directory called `build` like the rule, that's the only point of using PHONY), then any rule that depends on `build` (e.g. `all`) will be phony too. That's my understanding.